### PR TITLE
MWAA doc fix

### PR DESCRIPTION
**Pull request by season1946**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/awslabs/data-on-eks/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR to add a new Add-on for [Terraform EKS Blueprints](https://github.com/aws-ia/terraform-aws-eks-blueprints) repo (if applicable)
- [ ] Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: schedulers/managed-airflow-mwaa/main.tf [hunk 1]
```diff
@@ -149,7 +149,18 @@ locals {
...
+          command = "aws"
+          args = [
+            "--region",
+            local.region,
+            "eks",
+            "get-token",
+            "--cluster-name",
+            local.name
+          ]
+        }
```
Comment (vara-bonthu) [inline schedulers/managed-airflow-mwaa/main.tf:163]: Does it fail without using `exec` ? If not we will keep the data source `data.aws_eks_cluster_auth.this.token`

**Review by vara-bonthu — COMMENTED** [review]
LGTM! One minor comment

File: schedulers/managed-airflow-mwaa/main.tf (hunk 1 above)
Comment (season1946) [reply schedulers/managed-airflow-mwaa/main.tf:163]: @vara-bonthu the token works but will expire after 1h. I go to MWAA eks doc and find they will the above way. I've tested. thanks.
https://docs.aws.amazon.com/mwaa/latest/userguide/mwaa-eks-example.html#eksctl-kube-config

**Review by vara-bonthu — APPROVED** [review]
LGTM

**Pull request merged**